joomla/joomla-cms
# Snyc install sql for the Backend menus [#__menu]

Pull Request for Issue raised by @rdeutz 

### Summary of Changes

This syncs the backend main menu's over the testing data

### Testing Instructions

- Install all testing data via this package https://github.com/zero-24/joomla-cms/archive/fix_install_sql.zip
- make sure the backend menu's is complete 


### Documentation Changes Required

None
